pkg/providers/providersutils.go

```diff
@@ -277,8 +282,34 @@ func GetCurrentAzSubscriptionId() []string {
 		log.Fatal(err)
 	}
 
-	var ids []string
-	json.Unmarshal(out, &ids)
+	var id []string
+	json.Unmarshal(out, &id)
+
+	return id[0]
+}
+
+func GetAzSubscriptionLabels(id string) []SubLabel {
+	CheckAzCliInstalled()
+	if !IsLoggedInToAz() {
+		if err := LogInToAz(); err != nil {
+			log.Fatal(err)
```

Comment:
💡 suggestion please: (More informative error moving forward, I know at many places it just `err` 

```*.go
        CheckAzCliInstalled()
	if !IsLoggedInToAz() {
		if err := LogInToAz(); err != nil {
			log.Fatalf("Failed to log in to Azure: %v", err)
		}
	}

```